gcalcli/gcalcli.py

```diff
@@ -414,7 +416,24 @@ def _get_week_events(self, start_dt, end_dt, event_list):
                 if force_now_marker:
                     event_color = self.options['color_now_marker']
                 else:
-                    event_color = self._calendar_color(event['gcalcli_cal'])
+                    if self.options['override_color'] and event.get('colorId'):
```

Comment:
Ideally, we could structure this so that the override option is handled in the _calendar_color method.